packages/paymaster/src/utils/Types.ts

```diff
@@ -112,3 +112,20 @@ export enum PaymasterMode {
   ERC20 = "ERC20",
   SPONSORED = "SPONSORED",
 }
+
+// Converted to JsonRpcResponse with strict type
+export type EstimateUserOpGasResponse = {
+  jsonrpc: string;
+  id: number;
+  result: UserOpGasResponse;
+  error?: JsonRpcError;
+};
+
+export type UserOpGasResponse = {
```

Comment:
why don't we just edit above?
also if there will be new api version we wouldnt know which one is being called and what to expect in response

Reply:
updated